hack/make-bundle-vars.mk

```diff
@@ -137,6 +137,11 @@ IBM_ODF_SUBSCRIPTION_PACKAGE ?= ibm-storage-odf-operator
 IBM_ODF_SUBSCRIPTION_CHANNEL ?= stable-v1.7
 IBM_ODF_SUBSCRIPTION_CSVNAME ?= $(IBM_ODF_SUBSCRIPTION_PACKAGE).$(IBM_ODF_BUNDLE_VERSION)
 
+ODF_SNAPSHOT_CONTROLLER_BUNDLE_IMG ?= quay.io/ocs-dev/snapshot-controller-bundle:latest
```

Comment:
Lets not use the latest tag, It creates problems when the new image is pushed on the same tag.